### PR TITLE
Koreanfilter/new

### PR DESCRIPTION
**Pull request by ghost**
dcinside.com :
https://nstatic.dcinside.com/dc/dcad/w/ad_guide_pc2.html 
[2020_dcinside_ad.pdf](https://github.com/AdguardTeam/AdguardFilters/files/5899249/2020_dcinside_ad.pdf)
[2020_dcinside.pdf](https://github.com/AdguardTeam/AdguardFilters/files/5899248/2020_dcinside.pdf)

naver.com :
https://business.naver.com/service.html#section_02
https://displayad.naver.com/advertisement
https://saedu.naver.com/naverad/intro.nhn
https://gfa.naver.com/
https://join.shopping.naver.com/ad/theme/shopping_box.nhn
[DA_PLAT_Guide_1.pdf](https://github.com/AdguardTeam/AdguardFilters/files/5899261/DA_PLAT_Guide_1.pdf)
[Naver_Display_AD.pdf](https://github.com/AdguardTeam/AdguardFilters/files/5899262/Naver_Display_AD.pdf)

nate.com : 
https://adguide.nate.com/
https://keywordshop.nate.com/
[nate_displayAD.pdf](https://github.com/AdguardTeam/AdguardFilters/files/5899673/nate_displayAD.pdf)

zum.com : 
https://biz.zum.com/search
https://biz.zum.com/display/basic/1
https://biz.zum.com/commerce/1

inven.co.kr inven.net : 
http://www.inven.co.kr/partner/
http://www.inven.co.kr/ad/contact/
[INVEN-20200730-AD+.pdf](https://github.com/AdguardTeam/AdguardFilters/files/5901747/INVEN-20200730-AD%2B.pdf)
[INVEN-20200820-IT_BRAND_PARTNER_PROPOSAL-ENG.pdf](https://github.com/AdguardTeam/AdguardFilters/files/5901749/INVEN-20200820-IT_BRAND_PARTNER_PROPOSAL-ENG.pdf)

ilbe.com :
https://ad.ilbe.com/sponsor_content
https://ad.ilbe.com/new_sky

### CODE REVIEW

**Comment by piquark6046** [comment]
a4e319e048fe882f89aad07bf56ce6d49300250b
https://www.ytn.co.kr/_ln/0103_202101312154352147

**Comment by piquark6046** [comment]
bfe44f1907cb6f45fd601b6904e57b885f286847
https://biz.heraldcorp.com/view.php?ud=20210131000291

**Comment by piquark6046** [comment]
cb147700d5ea6e746c2aa53b61b0e73a935ea831
https://zum.com/#!/home

**Comment by piquark6046** [comment]
8341413c1c462289ca29a919754dd4b6f2fa5fe6
https://search.zum.com/search.zum?method=uni&option=accu&rd=1&qm=f_typing.top&query=%EC%B4%88%EB%B0%A5

**Comment by piquark6046** [comment]
dcb2a7cb106e9570c6cf0ea5bd466bda46b4c778
https://arca.live/b/dogdrip/20654817?p=1

**Comment by piquark6046** [comment]
7bb2cf38a77345857b894965eb59f4b8be17f852
https://www.dogdrip.net/305174448

**Comment by piquark6046** [comment]
bb85ad9bb260f1c984668d5bf1779afaf0be93c5
http://www.todayhumor.co.kr/board/view.php?table=gomin&no=1786718&s_no=15114745&kind=total&page=1

**Comment by piquark6046** [comment]
c0934d3c5a872237349b4a3d348330e7f557612e
http://web.humoruniv.com/board/humor/read.html?table=pds&pg=0&number=1035272

**Comment by piquark6046** [comment]
be4a558cfea0b7dedc6aa271d818e516be848e1a, cfb96fdc0686e663c06139e7faded094a7082257
http://www.inven.co.kr/webzine/news/?news=250969
http://www.inven.co.kr/
http://www.inven.co.kr/board/webzine/4262/6145?iskin=webzine

**Comment by piquark6046** [comment]
ca060023c3e6309429596f3873c74a7a66612665
https://namu.wiki/w/%EA%B0%9C%EB%93%9C%EB%A6%BD%EB%84%B7

**Comment by piquark6046** [comment]
f1edeb741baeceec978efeb4a3e5e6be1c37b37e
https://www.clien.net/service/
https://www.clien.net/service/board/news/15852060

**Comment by piquark6046** [comment]
2a85002dabf37d3d9f1ce7c879c645160676079c
https://www.ruliweb.com/
https://bbs.ruliweb.com/hobby/board/300117/read/30627941
https://bbs.ruliweb.com/news/read/146836

**Comment by piquark6046** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/73641/commits/de756af29899088e973e65d4c8e94a90d2622939, https://github.com/AdguardTeam/AdguardFilters/pull/73641/commits/c1e7a04d6df3ee27f449f4682791e367077bf6c0, https://github.com/AdguardTeam/AdguardFilters/pull/73641/commits/03f25d430c2f6752d432d1f97288178c387d4d60
https://gall.dcinside.com/board/view/?id=superidea&no=240566

**Comment by piquark6046** [comment]
3865c24cdcd3000c782951451766c11789f12885
https://www.yonhapnewstv.co.kr/news/MYH20210201017800038

**Comment by piquark6046** [comment]
a39f8353b8bdb90ec841b8c58e59fce53c21e806
https://www.donga.com/
https://www.donga.com/news/Society/article/all/20210201/105227247/2?ref=main

**Comment by piquark6046** [comment]
59fce1e4af57743602e9e66134fe1b26fa7d05d3
https://coding-factory.tistory.com/486

**Comment by piquark6046** [comment]
273245cdcc0a428b0424dabc5705d62941d95339
https://quasarzone.com/bbs/qb_free/views/3245424

**Comment by piquark6046** [comment]
42a2d67e79b88d244635caa8903a5d4a471e942b
AdguardTeam/AdguardFilters#73714

**Comment by piquark6046** [comment]
f6a65df96c86627d93e080e9e669602e8e408057
AdguardTeam/AdguardFilters#73959

**Comment by piquark6046** [comment]
f9e27026e25b46c3e541b0a2e1354820499ab1b6
AdguardTeam/AdguardFilters#74058

**Comment by piquark6046** [comment]
341b9e4735fbfbf0d13f74261515d879e8f1d3f4
https://cdmanii.com/6999

**Comment by piquark6046** [comment]
@wout-codes 
Your request is merged at 49321418fb17a7f31232f4bc8506a8f4ba491154. Thanks.

**Comment by wout-codes** [comment]
@piquark6046 Are you trying launch a new Korean Filter in AdGuard?

**Comment by piquark6046** [comment]
> 
> 
> @piquark6046 Are you trying launch a new Korean Filter in AdGuard?

Maybe. :D

**Comment by wout-codes** [comment]
> > @piquark6046 Are you trying launch a new Korean Filter in AdGuard?
> 
> Maybe. :D

I like the idea TBH. Keep up!

**Comment by piquark6046** [comment]
Anti-AdBlock Script:
https://xetown.com/tips/1128870

**Comment by wout-codes** [comment]
> Anti-AdBlock Script:
> https://xetown.com/tips/1128870

I see. Anything I can do?

**Comment by piquark6046** [comment]
> 
> 
> > Anti-AdBlock Script:
> > https://xetown.com/tips/1128870
> 
> I see. Anything I can do?

It is just a comment for leaving a URL about a script library killing the ad-block script.
If you want to block that, please write script and comment.
After that, I will commit it with marking you as a co-author.

**Comment by wout-codes** [comment]
Ok I will see about it. I am busy myself ;(

**Comment by Alex-302** [comment]
Guys, what about to commit Korean sites to List-KR?

**Comment by wout-codes** [comment]
I think if AdGuard has Korean Filter it would worth it.

**Comment by wout-codes** [comment]
List KR doesn't block anti-adblock if I am not mistaken.

**Comment by piquark6046** [comment]
> 
> 
> Guys, what about to commit Korean sites to List-KR?

Hmm... Does not AdGuard support a filter for Korean website?
If so, it will be permanently or temporarily?

**Comment by piquark6046** [comment]
2dba88e96920ad09656a20766bd0477e9cd87459
https://www.dogdrip.net/computer/109032174

**Comment by piquark6046** [comment]
788ef16d7c1d40448d72fb1a5a198a2e7eb67242
#74259

**Comment by piquark6046** [comment]
393466aca27d056ff8b72aca67b4841d82214850
https://doctorson0309.tistory.com/850

**Comment by piquark6046** [comment]
0defb1835f66a0975cd4982e8fdf1122e08447b1
https://arca.live/b/dogdrip/?mode=best

**Comment by piquark6046** [comment]
0102b4ceb89f53757a8ceac1e2864da7f220b508
https://bbs.ruliweb.com/hobby/board/300117/read/30628308
https://bbs.ruliweb.com/news/read/147063

**Comment by piquark6046** [comment]
268c4414feafaea84e1c74847ff2455beb1f8fb1
https://nhj12311.tistory.com/436?category=774995

**Comment by piquark6046** [comment]
b83edbb3741a1cad189981321190a41b2ab3392f
https://news.naver.com/
https://news.naver.com/main/read.nhn?mode=LSD&mid=shm&sid1=105&oid=029&aid=0002654166
https://news.naver.com/main/main.nhn?mode=LSD&mid=shm&sid1=105

**Comment by piquark6046** [comment]
3445ee2f1a76565839f5ab064cc6afa8aaf9bd5c
https://entertain.naver.com/home
https://entertain.naver.com/read?oid=109&aid=0004350812

**Comment by piquark6046** [comment]
f73a6a90e930a7b41315cf31d408c247b3d47b84
https://tv.naver.com/
https://tv.naver.com/v/18359490/list/67096

**Comment by wout-codes** [comment]
@piquark6046 Remove this, `||adshort.media^$document,popup` already added just now to the base filter.

**Comment by piquark6046** [comment]
> 
> 
> @piquark6046 Remove this, `||adshort.media^$document,popup` already added just now to the base filter.

Please see [this page](https://github.com/AdguardTeam/AdguardFilters/pull/73641/files).

**Comment by piquark6046** [comment]
6d272931219bf5bd654d2335ea1d99f9881b44d2
https://www.ilbe.com/list/jjal
https://www.ilbe.com/view/11322951190

**Comment by piquark6046** [comment]
de3ac5d4e8fb8fa6808332d5d8341508fddc5efa
#74567

**Comment by piquark6046** [comment]
49cdb008b9be455771a69de1360223ede9f2b53b
#74566

**Comment by piquark6046** [comment]
853285ed973008072f8d55c46558949f7ddc1a72
https://pann.nate.com/talk/357550580

**Comment by piquark6046** [comment]
d137e563669886edfc8d0713d6dce2feb44e6942
#74684

File: JapaneseFilter/sections/adservers.txt [hunk 1]
```diff
@@ -63,6 +63,7 @@
 ! Eimg.jp
 ||adingo.jp.eimg.jp^
 !
+||popin.cc^$third-party
```
Comment (Yuki2718) [inline JapaneseFilter/sections/adservers.txt:66]: `popin.cc` was intentionally unblocked, see https://github.com/AdguardTeam/AdguardFilters/issues/62998. There were even more discussion internally. We block popin ads in ad-blocking filter and tracker in Spyware as long as doing so doesn't cause trouble, but not entire popIn.

**Comment by Yuki2718** [comment]
Adding a new filter requires various changes in FiltersRegistry including translation, it's not very straight forward (I'm not opposing).

File: JapaneseFilter/sections/adservers.txt (hunk 1 above)
Comment (piquark6046) [reply JapaneseFilter/sections/adservers.txt:66]: Okay.
I will remove these rules.

Comment (piquark6046) [reply JapaneseFilter/sections/adservers.txt]: 99af4e5094e34aba402a8861ea2789f3c598e9a1

**Comment by piquark6046** [comment]
> 
> 
> Adding a new filter list requires various changes in FiltersRegistry including translation, it's not very straight forward (I'm not opposing). Of note, when AG adds a new list it's usually based off of a popular list as such AG Chinese is EL China + AG's own part. If you have a recommendation as a base of the new list, that'll be very appreciated - currently AG includes two Korean lists.

I knew what Youslist is under CC BY 4.0 on a moment ago.
But, List-KR is under CC BY-SA 4.0;

**Comment by Yuki2718** [comment]
> 
> I knew what Youslist is under CC BY 4.0 on a moment ago.

You mean now not? Both BY 4.0 and BY-SA 4.0 will be compatible to AG lists.

**Comment by piquark6046** [comment]
> 
> 
> > I knew what Youslist is under CC BY 4.0 on a moment ago.
> 
> You mean now not? Both BY 4.0 and BY-SA 4.0 will be compatible to AG lists.

Okay.

**Comment by piquark6046** [comment]
@Yuki2718 
According to description of [List-KR](https://github.com/List-KR/List-KR), It contains ad blocking, annoyance blocking and privacy protection function.
If you want to use this filter for only blocking ads, I don't recommend.
> This filter aims to filter not only ads, but also known trackers and other scripts which negatively affects user experience(including anti-adblock, automatic scrolling, and disabling context menu).

**Comment by piquark6046** [comment]
cf4af1f083cb5898a20b7ede08135b6895a6204a
https://www.etoland.co.kr/
https://www.etoland.co.kr/bbs/board.php?bo_table=etohumor04&wr_id=343360
#74875

**Comment by piquark6046** [comment]
39414dc20a3bb12279ca72a0f0a90824ba7aae50
https://m.comic.naver.com/webtoon/detail.nhn?titleId=753858&no=1&week=mon&listSortOrder=DESC&listPage=1

**Comment by piquark6046** [comment]
c930bdfabc744745ce6445ba563db967eb32dbf7
#74871

**Comment by piquark6046** [comment]
6c51faa10e62caadef85ce52dd7bc2438a8ded74
https://kin.naver.com/qna/detail.nhn?d1id=1&dirId=1060106&docId=382085422

**Comment by piquark6046** [comment]
3338754a34df369ba9c2475dacb2d2233a8acc6f
https://m.ruliweb.com/community/board/300143/read/50982287
https://m.ruliweb.com/
https://m.ruliweb.com/news/529/read/147478

**Comment by piquark6046** [comment]
5465fd5414aeb90c564f65fee817a3359b97b212
https://adop.cc/en/home-2/

https://m.ruliweb.com/community/board/300143/read/50982287
https://m.ruliweb.com/news/529/read/147478

**Comment by Alex-302** [comment]
@piquark6046 Hi. Why don't you want to create filter in your own repository? We cannot just merge it.
Or maybe work jointly with YousList or List-KR?

**Comment by piquark6046** [comment]
> 
> 
> @piquark6046 Hi. Why don't you want to create filter in your own repository? We cannot just merge it.
> Or maybe work jointly with YousList or List-KR?

Okay.

**Comment by Alex-302** [comment]
If it will be a separate filter or addition to YousList/List-KR, we can include it to our filters list.

**Pull request closed without merging**